### PR TITLE
Refactoring cron jobs

### LINKED ISSUE #153: Combine cron jobs
# Background

Right now we have 2 cron jobs running at 8am PST and 1 cron job running at 12am PST. We should consolidate these jobs - run them at the same time - and reduce all functions into one scheduled job.
image

# Task

 - Create a function that calls all 3 jobs we currently have in place
 - Replace repeated cron scheduling code with one job that calls the combined function
 - Move all individual functions (the ones that were combined into the combined function) into /helpers

### PR DESCRIPTION
**Pull request by syd-tan**
# Refactoring cron jobs

Closes #153 

# Summary of change
Previously, there were three separate cron jobs running some at different times, now there the three functions are called in a helper function at cronJob.helper.js, so app.js runs that single cron job at 8:00 am PST.

# Testing/Verification

- Currently have the cron job set to every minute, so once you start up the backend, every minute or so you should see "running cron jobs" on the console, as well as some edits to housekeeping collections, "pinging test", and "finished cron jobs".

### CODE REVIEW

File: routes/user.js [hunk 1]
```diff
@@ -284,6 +284,7 @@ router.post("/acceptFriendRequest", requireUserAuth, async (req, res) => {
       { _id: friendId },
       { $push: { friends: userId } }
     );
+    friend.outgoingFriendRequests.pull = userId;
```
Comment (aidenszeto) [inline routes/user.js:287]: We can remove this line, think it might be an artifact from rebase

**Review by aidenszeto — APPROVED** [review]
One thing we could discuss/consider is moving `findExpiringListings` from `expiredListings.helper.js` to `cronJobs.helper.js` to cut down on the extra import. Either way should work but changes look good to me!

File: routes/user.js (hunk 1 above)
Comment (syd-tan) [reply routes/user.js]: Cool I took that out and as for combining the expiredListings.helper to cronJobs.helper we could probably do that during our refactoring sprint

Comment (aidenszeto) [reply routes/user.js]: Good idea, everything else looks good to me!

File: routes/user.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ const mongoose = require("mongoose");
 const User = require("../models/user.model");
 const { requireUserAuth } = require("../utils");
 const { multerUploads, uploadImagesToAWS } = require("./photos");
-const {sendVerificationEmail} = require('../helpers/account.helper')
+const { sendVerificationEmail } = require("../helpers/account.helper");
```
Comment (nishantbalaji) [inline routes/user.js:7]: the emails are now all in `../helpers/emails.helper` btw

**Review by nishantbalaji — CHANGES_REQUESTED** [review]

File: routes/user.js (hunk 2 above)
Comment (syd-tan) [reply routes/user.js]: Fixed!

**Review by nishantbalaji — APPROVED** [review]

**Pull request merged**